src/CHANGELOG.mdx

```diff
@@ -5,6 +5,12 @@ route: /changelog
 
 # Changelog
 
+#### [v. TODO]
```

Comment:
I would either delete this, or UP the version in `package.json`, write the version here with your name below